src/index.js

```diff
@@ -0,0 +1,337 @@
+import { LanguageEN } from "./En";
+import { LanguageRU } from "./Ru";
+import "./style.css";
+
+const buttonRefresh = document.querySelector("#control_button");
+const buttonFarenheit = document.querySelector("#farenheit");
+const buttonCelsius = document.querySelector("#celsius");
+const inputCity = document.querySelector(".search_input");
+const buttonSearch = document.querySelector("#search_button");
+const locationCity = document.querySelector("#location_city");
+const dateNow = document.querySelector(".date_now");
+const timeNow = document.querySelector(".time_now");
+const tempretureNow = document.querySelector(".tempreture_now");
+const latitude = document.querySelector(".latitude");
+const longitude = document.querySelector(".longitude");
+const body = document.querySelector("#body");
+const overcast = document.querySelector("#overcast");
+const feelsLike = document.querySelector("#feelsLike");
+const speedWind = document.querySelector("#speedWind");
+const humidity = document.querySelector("#humidity");
+const iconWeatherNow = document.querySelector(".icon_weatherNow");
+const iconOne = document.querySelector(".icon_one");
+const iconTwo = document.querySelector(".icon_two");
+const iconThree = document.querySelector(".icon_three");
+const firstDay = document.querySelector("#firstDay");
+const secondDay = document.querySelector("#secondDay");
+const thirdDay = document.querySelector("#thirdDay");
+const firstTemperature = document.querySelector("#firstTemperature");
+const secondTemperature = document.querySelector("#secondTemperature");
+const thirdTemperature = document.querySelector("#thirdTemperature");
+const buttonRussianLanguage = document.querySelector("#language_ru");
+const buttonEnglishlanguage = document.querySelector("#language_en");
+const localLang = localStorage.getItem("lang");
+const isRu = localLang && localLang === "ru";
+let info = isRu ? LanguageRU : LanguageEN;
+
+// button Language RU and EH
+function activeButtonLang(buttonRussianLanguage, buttonEnglishlanguage) {
+    buttonRussianLanguage.classList.add("active");
+    buttonRussianLanguage.classList.remove("not_active");
+    buttonEnglishlanguage.classList.remove("active");
+    buttonEnglishlanguage.classList.add("not_active");
+};
+
+function initializeLangButton() {
+    if (isRu) {
+        activeButtonLang(buttonRussianLanguage, buttonEnglishlanguage);
+    }
+};
+initializeLangButton();
```

Comment:
please move this logic to initializeCity

Reply:
I'm move this logic to setLang